client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -349,7 +351,8 @@ export default function CompositeCheckout( {
 		stripeConfiguration,
 		stripe,
 		isApplePayAvailable,
-		isApplePayLoading,
+		isGooglePayAvailable,
```

Comment:
This property seems to be unused currently?  (Maybe it will be used in a later pull request, though?)

Reply:
Yeah, sorry about that. This PR was extracted from that one, and some of the rebasing was tricky so I had left some minor google-specific stuff in.